scripts/prepare_gcp_enclave_metadata.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
```

Comment:
It might be worth organising the gcp specific files into their own folder, as this folder will get busy once we add azure and aws tests. So putting the files into /scripts/gcp might make sense?